src/node_external_reference.h

```diff
@@ -47,6 +47,9 @@ using CFunctionCallbackWithUint8ArrayUint32Int64Bool =
                 uint32_t,
                 int64_t,
                 bool);
+using CFunctionWithObjectInt32Fallback = void (*)(v8::Local<v8::Object>,
+                                                  const int32_t input,
```

Comment:
`const` is not part of the signature.

```suggestion
                                                  int32_t input,
```

Reply:
I don't follow?

Reply:
IIRC, the `const` keyword is not part of the function signature and hence not part of the function type being declared, so it should probably be removed.

Reply:
I'll remove it. Thanks!